eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/mutable/AbstractMutableBag.java

```diff
@@ -274,4 +275,12 @@ public MutableShortBag collectShort(ShortFunction<? super T> shortFunction)
         }
         return new NonParallelUnsortedBag<>(this);
     }
+
+    @Override
+    public MutableSet<T> selectUnique()
+    {
+        MutableBag<T> selected = this.selectByOccurrences(occurrences -> occurrences == 1);
```

Comment:
This can be made efficient by using forEachWithOccurrences() similar to the primitive side comments.